src/ctia/entity/relationship.clj

```diff
@@ -62,6 +71,69 @@
   (st/merge PagingParams
             RelationshipFieldsParam))
 
+(s/defschema IncidentCasebookLinkRequest
+  {:casebook_id Reference
+   (s/optional-key :tlp) TLP})
+
+(def incident-casebook-link-route
+  (POST "/:id/link" []
+        :return rs/Relationship
+        :body [link-req IncidentCasebookLinkRequest
+               {:description "an Incident Link request"}]
+        :header-params [{Authorization :- (s/maybe s/Str) nil}]
+        :summary "Link an Incident to a Casebook"
+        :path-params [id :- s/Str]
+        :capabilities #{:create-incident
```

Comment:
Why the `:create-incident` capability is required? Only a Relationship is created.